azure-pipelines.yml

```diff
@@ -223,6 +223,11 @@ jobs:
         NUSPEC="pack\Microsoft.Azure.Functions.V4.PythonWorker.nuspec"
         WKVERSION="$VERSION-$(patchBuildNumberForDev)"
       else
+        echo "Generating V3 Integration Test Package for $BUILD_SOURCEBRANCHNAME"
+        LATEST_TAG=$(curl https://api.github.com/repos/Azure/azure-functions-python-worker/tags -s | jq '.[0].name' | sed 's/\"//g' |  cut -d'.' -f-2)
+        NUSPEC="pack\Microsoft.Azure.Functions.V3.PythonWorker.nuspec"
+        # Only required for Integration Test. Version number contains date (e.g. 3.1.2.20211028-dev)
+        WKVERSION="3.$LATEST_TAG-$(BUILD_BUILDID)-TEST"
```

Comment:
Sorry - didn't understand this - why do we need to add this section? Ideally, this azure-pipelines now should run on v3.x-dev branch, and that would generate the versions required for V3.

Reply:
I agree the echo statement is bit confusing .. I added this to test any pipeline related task that we need to run from personal feature branch. I was not able to test the flow because the nuspec related settings were only from dev or v3/v4 branches, so I thought adding an else will help to test it easily. Do you have any other recommendation?